test/integration/params-validations-test.js

```diff
@@ -127,4 +127,22 @@ describe('params validations', () => {
       due_on: new Date('2012-10-09T23:39:01Z')
     })
   })
+
+  it.skip('Date is passed in correct format for notifications', () => {
+    const github = new GitHub()
+
+    nock('https://api.github.com/notifications')
```

Comment:
Adding this mock causes other tests to fail. However, it doesn't seem like the changes should happen in [octokit/fixtures](https://github.com/octokit/fixtures) since we need to inspect the query parameter as part of the test. 

I'm not too familiar with nock to know why this is causing failures or what the proper mocking should be